examples/langchain/README.md

```diff
@@ -0,0 +1,96 @@
+# ChatDKG Project README
+
+This example walks you through the process of creating a knowledge asset on the Origintrail decentralized knowledge graph and searching the knowledge asset using the power of Langchain and OpenAI.
```

Comment:
Let's provide some more context in the readme: what are KAs, reference Milvus, explain in a few steps what the algo looks like, include links where appropriate, including the video where we showed this demo (ideally at the point of showing it)